src/index.spec.ts

```diff
@@ -18,7 +18,12 @@ const SEM_VER_REGEX = /^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(0|[1-9]\d*|\
 function awsMock(): Aws {
   return {
     getStage: () => "dev",
-    request: (service, method, params: any) => Promise.reject("Log group doesn't exist"),
+    request: (service, method, params: any) => {
+      if (method == "getFunction") {
+        return Promise.resolve();
+      }
+      return Promise.reject("Log group doesn't exist");
```

Comment:
It seems a bit odd to have the Promise reject with "Log group doesn't exist" for lots of methods that don't have anything to do with log groups. Maybe instead of having `getFunction` be the special case, you can just have the rejection only occur when looking for a log group, and make the everything else resolve?